tests/routes.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require('supertest');
+const app = require('../server/index.js');
+
+describe('Tests all Crud Operations', () => {
+
+  // beforeAll(() => {
+
+  // });
+
+  // afterAll(() => {
+
+  // });
```

Comment:
What is this? I'm confused. Could you explain? //not mad

Reply:
I have a leak in my tests and I think this is apart of the solution, but I need to figure out what to put inside.